crates/client/Cargo.toml

```diff
@@ -55,6 +55,10 @@ url.workspace = true
 util.workspace = true
 worktree.workspace = true
 
+# fix for https://github.com/zed-industries/zed/actions/runs/11025748924/job/30632735188?pr=18315
+[package.metadata.cargo-machete]
+ignored = ["fs"]
+
```

Comment:
```suggestion
```

This is wrong, you should revert the commit and instead merge the `main`-branch into your branch, as #18324 was merged earlier and resolves this issue.

Reply:
ok, done